README.md

```diff
@@ -5,106 +5,108 @@
 
 The Format is:
 
 ```
-<base-encoding-character><base-encoded-data>
+<base-encoding-code-point><base-encoded-data>
 ```
 
-Where `<base-encoding-character>` is used according to the multibase table.
+Where `<base-encoding-code-point>` is a code representing an entry in the multibase table.
 
 ### Multibase Table
 
 The current multibase table is [here](multibase.csv):
 
 ```
-encoding,          code, description,                                                  status
-identity,          0x00, 8-bit binary (encoder and decoder keeps data unmodified),     default
-base2,             0,    binary (01010101),                                            candidate
-base8,             7,    octal,                                                        draft
-base10,            9,    decimal,                                                      draft
-base16,            f,    hexadecimal,                                                  default
-base16upper,       F,    hexadecimal,                                                  default
-base32hex,         v,    rfc4648 case-insensitive - no padding - highest char,         candidate
-base32hexupper,    V,    rfc4648 case-insensitive - no padding - highest char,         candidate
-base32hexpad,      t,    rfc4648 case-insensitive - with padding,                      candidate
-base32hexpadupper, T,    rfc4648 case-insensitive - with padding,                      candidate
-base32,            b,    rfc4648 case-insensitive - no padding,                        default
-base32upper,       B,    rfc4648 case-insensitive - no padding,                        default
-base32pad,         c,    rfc4648 case-insensitive - with padding,                      candidate
-base32padupper,    C,    rfc4648 case-insensitive - with padding,                      candidate
-base32z,           h,    z-base-32 (used by Tahoe-LAFS),                               draft
-base36,            k,    base36 [0-9a-z] case-insensitive - no padding,                draft
-base36upper,       K,    base36 [0-9a-z] case-insensitive - no padding,                draft
-base58btc,         z,    base58 bitcoin,                                               default
-base58flickr,      Z,    base58 flicker,                                               candidate
-base64,            m,    rfc4648 no padding,                                           default
-base64pad,         M,    rfc4648 with padding - MIME encoding,                         candidate
-base64url,         u,    rfc4648 no padding,                                           default
-base64urlpad,      U,    rfc4648 with padding,                                         default
-proquint,          p,    PRO-QUINT https://arxiv.org/html/0901.4016,                   draft
-base256emoji,      🚀,    base256 with custom alphabet using variable-sized-codepoints, draft
+code,   Unicode,    encoding,           description,                                                    status
```

Comment:
I'm unsure if "code" is the right word here. It's more like the "ASCII representation", so perhaps just "ASCII"? I would then move it into the second (column) as the Unicode is really the canonical thing.

Reply:
The goal is to communicate that the first column is the display name for the encoding and the second column is the machine readable prefix.

code => display name for humans
Unicode => machine readable prefix (runes)
utf-8 => machine readable prefix (bytes)

I don't know if these name communicate the conesepts well or not.

Reply:
In the newest version the columns are called:

Unicode => The thing that implementations should care about
Character => Display name what users see

Is that better?